inception-project/inception
# #5089 - Document structure is not retained when preparing a document as curation target

**What's in the PR**
* Transfer document structure when preparing the curation CAS
* Offer check for missing document structure in curation CAS
* Offer repair for missing document structure in curation CAS

**How to test manually**
* See issue description
* Try check and repair

**Automatic testing**

**Documentation**
